caikit/interfaces/runtime/data_model/training_management.py

```diff
@@ -43,7 +43,7 @@ class ModelPointer(DataObjectBase):
 @dataobject(RUNTIME_PACKAGE)
 class TrainingStatusResponse(DataObjectBase):
     training_id: Annotated[str, FieldNumber(1)]
-    state: Annotated[TrainingStatus, FieldNumber(2)]
+    state: Annotated[JobStatus, FieldNumber(2)]
```

Comment:
This will revert if we do the inheritance thing which I think would be good. I think this is _technically_ an API breaking change as is since the name of the message is changing.

Reply:
So inheritance thing is a no go until we fix the py-to-proto bug (I'll look into it later today). I'll leave it for now until that's fixed.

Reply:
Ok, yeah, I do want to be careful that we don't break the API by accident here

Reply:
Since protobuf sends enums as their values and not types it won't break anything over the wire; however, it will break people who are accessing the enum via lookups (like `DataBase.get_class_by_name` or people doing type checks (though the mapping should help this case). IMO it's probably worth a v27